recipes/mne-python/construct.yaml

```diff
@@ -63,90 +59,15 @@ specs:
   - wheel =0.45.1
   - conda =25.3.1
   - mamba =2.1.0
-  # MNE ecosystem
-  # TODO: ⛔️ ⛔️ ⛔️ DEV BUILDS START: CHANGE BEFORE RELEASE! ⛔️ ⛔️ ⛔️
-  # - mne-base =1.4dev0=*_20230503
-  # - mne-installer-menus =1.4dev0=*_20230503
-  # - mne-bids =0.11dev0=*_20221007
-  # TODO: ⛔️ ⛔️ ⛔️ DEV BUILDS STOP: CHANGE BEFORE RELEASE! ⛔️ ⛔️ ⛔️
-  - mne =1.9.0=pyside6_*_100
-  - mne-installer-menus =1.9.0=*_100
   # For testing purposes with build_local.sh, you can comment out all deps
   # below for speed, and change mne to mne-base above
 
-  # vvv this line is special and used by MNE-Python, do not change it!
+  # vvv this line is special and used by tests, do not change it!
+  # See: tests/test_imports.py
   # <<< BEGIN RELATED SOFTWARE LIST >>>
-  - mne-bids =0.16.0
-  - mne-bids-pipeline =1.9.0
-  - mne-qt-browser =0.7.1
-  - mne-connectivity =0.7.0=*_1
-  - mne-faster =1.2.2
-  - mne-nirs =0.7.1
-  - mne-features =0.3
-  - mne-rsa =0.9.1
-  - mne-ari =0.1.2
-  - mne-icalabel =0.7.0
-  - mne-gui-addons =0.1=*_1
-  - mne-lsl =1.9.0
-  - mne-kit-gui =1.3.0
-  - autoreject =0.4.3
-  - antio =0.5.0
-  - wfdb =4.3.0
-  - meegkit =0.1.9
-  - eeg_positions =2.1.2
-  # MRI
-  - fsleyes =1.14.2
-  - dcm2niix =1.0.20241211
-  - dipy =1.11.0
-  # Time-frequency analysis
-  - pactools =0.3.1
-  - tensorpac =0.6.5
-  - emd =0.8.1
-  - neurodsp =2.3.0
-  - bycycle =1.1.0
-  - fooof =1.1.1
-  # Microstates
-  - mne-microstates =0.3.0
-  - pycrostates =0.6.1
-  # OpenNeuro.org data access
-  - openneuro-py =2024.2.0
-  # sleep staging
-  - sleepecg =0.5.9=*_2
-  - yasa =0.6.5
-  # various biological signals (ECG, EOG, EMG, …)
-  - neurokit2 =0.2.10
-  - mnelab =1.0.0
-  # other
-  - pyriemann =0.8
-  - pyprep =0.4.3=*_1
-  - python-picard =0.8
-  - pybv =0.7.6
-  - edfio =0.4.7  # allow_outdated, for some reason 0.4.8 won't download
-  - eeglabio =0.0.3
-  - mffpy =0.10.0
-  - openmeeg =2.5.15
-  - python-neo =0.14.1
-  - nitime =0.11
-  - snirf =0.8.0
-  - alphacsc =0.4.1
-  # Not on PyPI or no wheels on PyPI
-  # conpy: https://aaltoimaginglanguage.github.io/conpy/
-  # posthoc: https://users.aalto.fi/~vanvlm1/posthoc/python/
-  # eelbrain: https://eelbrain.readthedocs.io/en/stable/index.html
-  # Not on conda-forge
-  # niseq: https://github.com/john-veillette/niseq
-  # sesameeg: https://pybees.github.io/sesameeg
-  # invertmeeg: https://github.com/LukeTheHecker/invert
-  # Meggie: https://github.com/cibr-jyu/meggie
-  # MEM: https://github.com/multifunkim/best-python
   # <<< END RELATED SOFTWARE LIST >>>
-  # ^^^ this line is special and used by MNE-Python, do not change it!
+  # ^^^ this line is special and used tests, do not change it!
```

Comment:
```suggestion
  # ^^^ this line is special and used by tests, do not change it!
```

Reply:
Removing all the "related packages" (which is an MNE-Python specific thing) is going to lead to an `assert` failure in `tests/test_imports.py` (currently it seems to be failing somewhere else earlier).  So I think we will want to purge these "special" lines too, and alter `tests/test_imports.py` to not look for them.

Reply:
Done - I've refactored the import script to check basically all packages in the spec list.